### ISSUE TITLE
Fix Typo

### ISSUE THREAD

**Issue by Luxovious** [RobTillaart/PCA9634#9]
Hi Rob,
I found a typo in the code. I have been wondering why the register address of the PWM-registers was defined as this:
`#define PCA9634_PWM(x)              (0x82+(x))`

The correct definition should be
`#define PCA9634_PWM(x)              (0x02+(x))`

However, as the MSB overflows, this does not cause any problems but it's technically incorrect and might confuse other users. It took me quiet a while to wrap my head around it. As 0x02 works as intendet I assume that this is just a "working typo".

Best regards Johannes

**Comment by Luxovious** [comment]
ah I forgot:
- FILE: PCA9634.h
- Line 19

**Comment by RobTillaart** [comment]
Thanks for pointing out. 

It is also in the PCA9635, so it is caused by the copy paste when I setup this library.

TODO: check datasheet for both libraries. 

done, 0x02 is the base register indeed.

The 0x80 part sets the auto-increment mode for all registers. 
It is a (minimal) comment in the .h file 

From paragraph 7.2 datasheet
_1 0 0 Auto-Increment for all registers. D[4:0] roll over to ‘0 0000’ after the last register (1 1011) is accessed._


it should allow faster writing / reading of the registers, but I do not recall using it explicitly 

Conclusion it is not a bug, it should be documented better.

**Comment by RobTillaart** [comment]
checked the code and the library uses the auto-increment in **writeN()** so it is even mandatory.
(amazing how fast one can forget details of your own code).

```cpp
uint8_t PCA9634::writeN(uint8_t channel, uint8_t* arr, uint8_t count)
{
  if (channel + count > _channelCount)
  {
    _error = PCA9634_ERR_WRITE;
    return PCA9634_ERROR;
  }
  uint8_t base = PCA9634_PWM(channel);
  _wire->beginTransmission(_address);
  _wire->write(base);
  for(uint8_t i = 0; i < count; i++)
  {
    _wire->write(arr[i]);            //  <<<<<<<<<<<<<<< here 
  }
  _error = _wire->endTransmission();
  if (_error != 0)
  {
    _error = PCA9634_ERR_I2C;
    return PCA9634_ERROR;
  }
  return PCA9634_OK;
}
```

I noticed there is no **readN()** function to read back the values set.
Do you have a need for that? (would be a new issue)

**Comment by RobTillaart** [comment]
done: improved comments in the .h file (develop branch)

**Comment by RobTillaart** [comment]
solved with 0.1.2
